xml/schema/CRM/Eventmessages/EventMessageRule.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
+
+<table>
+  <base>CRM/Eventmessages</base>
+  <class>EventMessageRule</class>
+  <name>civicrm_event_message_rules</name>
+  <comment>Event Messages Rule Entity</comment>
+  <log>true</log>
+
+  <field>
+    <name>id</name>
+    <type>int unsigned</type>
+    <required>true</required>
+    <comment>Unique EventMessageRule ID</comment>
+    <html>
+      <type>Number</type>
+    </html>
+  </field>
+  <primaryKey>
+    <name>id</name>
+    <autoincrement>true</autoincrement>
+  </primaryKey>
+
+  <field>
+    <name>event_id</name>
+    <type>int unsigned</type>
+    <required>true</required>
+    <comment>FK to Event</comment>
+  </field>
+  <foreignKey>
+    <name>event_id</name>
+    <table>civicrm_event</table>
+    <key>id</key>
+    <onDelete>CASCADE</onDelete>
+  </foreignKey>
+
+  <field>
+    <name>is_active</name>
+    <type>int unsigned</type>
+    <comment>FK to Event</comment>
```

Comment:
Should be the same as in the [current schema](https://github.com/systopia/de.systopia.eventmessages/blob/b057912f8db70be3cdeef3361cd9933a87cb5f43/sql/civicrm_value_event_messages.sql#L5C73-L5C73): `is this rule active`.